cwms-data-api/src/test/java/cwms/cda/api/VersionedTimeseriesControllerTestIT.java

```diff
@@ -0,0 +1,664 @@
+package cwms.cda.api;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletResponse;
+
+import cwms.cda.api.enums.VersionType;
+import cwms.cda.formatters.Formats;
+import fixtures.TestAccounts;
+
+import io.javalin.core.validation.JavalinValidation;
+import io.restassured.filter.log.LogDetail;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@Tag("integration")
+public class VersionedTimeseriesControllerTestIT extends DataApiTestIT {
+
+    private static final String OFFICE = "SWT";
+    private static final String locationId = "TsVersionedTestLoc";
+    private static final String tsId = locationId + ".Flow.Inst.1Hour.0.raw";
+    private static final String BEGIN_STR = "2008-05-01T15:00:00Z";
+    private static final String END_STR = "2008-05-01T17:00:00Z";
+
+    private Map<String, String> createParamMap() {
+        return new HashMap<String, String>()
```

Comment:
Nice trick. The downstream test code looks very clean.

Reply:
Thanks!

Reply:
I agree it looks clean but I think I prefer seeing all the params in each test.  I don't want to have to spend the mental energy to find where the map is created and what else may have changed it between creation and use.

Reply:
I should add that I don't care strongly enough to have you change the pr.  Just pointing out opinions.